### ISSUE TITLE
Display user query in search text box

### ISSUE THREAD

**Issue by mmulich** [Connexions/webview#188]
tldr;
After a search is done, the query string should be displayed in the search text box.

(More details on the data being sent from cnx-archive...)
The parsed query string can be found in the returned query object as `query.limits`. This resulting value from cnx-archive is a list of parsed terms. For example, if the user types `"nuclear physics" author:'Bill Nye' type:book sort:pubDate` then the `query.limits` value will be `[{"text": "nuclear physics"}, {"author": "Bill Nye"}, {"type": "book"}]`. Note that the `sort:pubDate` does not appear in this list. It can be found at `query.sort` as `pubDate`.
